Gemfile

```diff
@@ -29,6 +30,13 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+#doing the following to enable https://rails-assets.org/#/
+source 'http://insecure.rails-assets.org' do
+  gem 'rails-assets-bootstrap'
+  gem 'rails-assets-angular'
+  gem 'rails-assets-leaflet'
```

Comment:
So we don't actually want `rails-assets-angular` and `rails-assets-leaflet` as dependencies. Just `bootstrap`.